components/Dogs.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react'
+import { View, Text, ActivityIndicator } from 'react-native'
+
+import styled from 'styled-components/native'
+import { Accelerometer } from 'expo-sensors'
+import { useFonts, Montserrat_500Medium } from '@expo-google-fonts/montserrat'
+
+const ScreenBackground = styled.ImageBackground`
+	height: 100%;
+	resize-mode: center;
+`
+const MainView = styled.View`
+	display: flex;
+	align-items: center;
+`
+const MovieView = styled.View`
+	display: flex;
+	justify-content: center;
+	flex-direction: row;
+	width: 100%;
+`
+const Animation = styled.Image`
+	width: 150px;
+	height: 150px;
+`
+const MovieImage = styled.Image`
+	width: 340px;
+	height: 340px;
+	resize-mode: center;
+`
+const QuoteText = styled.Text`
+	padding: 3px;
+	font-weight: bold;
+`
+
+const Dogs = () => {
+	const [data, setData] = useState({
+		x: 0,
+		y: 0,
+		z: 0,
+	})
+	const [dog, setDog] = useState([])
+	const [quote, setQuote] = useState({})
+	const [loading, setLoading] = useState(false)
+	const [subscription, setSubscription] = useState(null)
+	const [fontsLoaded] = useFonts({
```

Comment:
Nice made with google fonts. Is there a way to get that font everywhere? I notice now that u have to add it on each style.